### PR TITLE
Add review modal of permissionless pool

### PR DESCRIPTION
**Pull request by nkrulikovsky**
Review modal of create permissionless pool.

image

### CODE REVIEW

**Comment by nkrulikovsky** [comment]
@ug02fast   the theme of `Alert` is updating on the `ReviewMigrationPool.tsx` update. So, I believe after merging that branch it could update automatically.

File: src/pages/CreatePool.tsx
Comment (ug02fast) [inline src/pages/CreatePool.tsx:182]: Could you also hook this up with component state. This part isn't dependent on any web3 interactions.

**Review by ug02fast — CHANGES_REQUESTED** [review]
Looks good just a couple requests. As this page component is gearing to become too large after functionality is added in, in the future can you break this page's mUI components into their own individual components and abstract it to some degree? Possible instead of having CreatePool.tsx have it be a directory with an index and a sub-directory full of components that make up the CreatePool page.

**Comment by ug02fast** [comment]
@hammeiam do we need text translated here?

File: src/pages/CreatePool.tsx
Comment (nkrulikovsky) [reply src/pages/CreatePool.tsx:185]: We necessary this props because originally Alert component is having icon and if we give false to icon props, then the icon don't appear

Comment (ug02fast) [reply src/pages/CreatePool.tsx:185]: I don't see that pattern often. You're right though.

File: src/pages/CreatePool/CreatePoolDialog.tsx
Comment (ug02fast) [inline src/pages/CreatePool/CreatePoolDialog.tsx:18]: what does defaulting to `true` do here?

File: src/pages/CreatePool/CreatePoolDialog.tsx
Comment (ug02fast) [inline src/pages/CreatePool/CreatePoolDialog.tsx:69]: Can you hook this up to close modal when clicked?

File: src/pages/CreatePool/CreatePoolDialog.tsx
Comment (ug02fast) [inline src/pages/CreatePool/CreatePoolDialog.tsx:25]: What does `onClose` on the `DialogTitle` do?

**Review by ug02fast — APPROVED** [review]
lgtm if onClose is working correctly. thanks for breaking up the files

File: src/pages/CreatePool/CreatePoolDialog.tsx
Comment (nkrulikovsky) [reply src/pages/CreatePool/CreatePoolDialog.tsx:25]: It close modal.

File: src/pages/CreatePool/CreatePoolDialog.tsx
Comment (nkrulikovsky) [reply src/pages/CreatePool/CreatePoolDialog.tsx:18]: Oh, that was testing. removed.

File: src/pages/CreatePool/CreatePoolDialog.tsx
Comment (ug02fast) [reply src/pages/CreatePool/CreatePoolDialog.tsx:29]: Clicking on the DialogTitle closes it? onClose handler should be on `Dialog`.

Comment (nkrulikovsky) [reply src/pages/CreatePool/CreatePoolDialog.tsx:29]: OK, let me do it on another branch.

File: src/pages/CreatePool/index.tsx
Comment (nkrulikovsky) [reply src/pages/CreatePool/index.tsx:185]: without `div`, we are having t
Screen Shot 2022-03-21 at 11 18 06 PM
he following error.

File: src/pages/CreatePool/index.tsx
Comment (nkrulikovsky) [reply src/pages/CreatePool/index.tsx]: Yeah, that would be good.
Updated!

File: src/pages/CreatePool/index.tsx
Comment (nkrulikovsky) [reply src/pages/CreatePool/index.tsx]: Updated!

**Pull request merged**
